plans/759/project-pages-ledger.md

```
@@ -709,6 +709,390 @@ The return type at `:17` is `{'outstanding'|'partial'|'settled'|'overpaid'|null}
+
+```bash
+S=d70aa8ac9fca414777985bb7dc74faa0462690e6; cd ~/GitHub/friends-and-family-billing
+git grep -n 'rawToken' "$S" -- src/lib/share.js src/lib/ShareLinkService.js src/app/views/Manage/InvoicingTab.jsx
+git show "${S}:firestore.rules" | sed -n '54,68p'
+```
+
+### C29—what the read rule actually permits
+
+**SPLIT: "the read is bounded by token possession" is SUPPORTED; "the boundary is enforced by the security rules" is WRONG—the rule bounds nothing beyond knowing the document id, and it grants `list` as well as `get`.** `firestore.rules:73-74`, verbatim:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)